packages/app/src/components/drawer/reuse-app-drawer.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Box, Flex, Stack, Button, Text } from '@blockstack/ui';
+import { ScreenBody, ScreenActions } from '@screen';
+import { Title } from '@components/typography';
+import { BaseDrawer, BaseDrawerProps } from './index';
+
+import { Image } from '@components/image';
+import { ConfigApp } from '@stacks/keychain';
+
+interface PreviousAppsProps {
+  apps: ConfigApp[];
+}
+
+const PreviousApps = ({ apps, ...rest }: PreviousAppsProps) => (
+  <Flex mx={6} {...rest}>
+    {apps.map((app, key) => (
+      <Box border="2px solid white" size="26px" borderRadius="6px" key={key} overflow="hidden">
+        <Image src={app.appIcon} alt={app.name} title={app.name} />
+      </Box>
+    ))}
+  </Flex>
+);
+
+interface ReuseAppDrawerProps extends BaseDrawerProps {
+  apps: ConfigApp[];
+  confirm: (hideWarning: boolean) => Promise<void>;
```

Comment:
Could flip to the positive, `showWarning`

Reply:
It's intended to match the value of the checkbox - when you check the box, you're saying "hide this warning in the future". So "checked" = "hide".